docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: '3'
+
+networks:
+  db-data-net:
+    driver: bridge
+
+volumes:
+ db-data:
+
+services:
+  mysql:
+    image: my-sql:1.2
```

Comment:
The image name `my-sql:1.2` seems unusual as it doesn't match the standard MySQL image naming convention. Ensure that this is the intended image or consider using the official MySQL image like `mysql:latest`.